index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leading Eagle Brand</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="https://use.typekit.net/xxz1eii.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+</head>
+
+<body>
+    <header>
+
+        <nav class="navbar">
+            <div class="navbar-innercontainer ">
+                <a class="brand" href="/index.html"><img src="/Assets/Logos/navlogo.png" class="nav-logo"
+                        alt="nav-logo"></a>
+                <ul class="nav-items" id="nav-menu">
+                    <li class="nav-link" id="navlink"><a href="/index.html" class="link active">Home</a></li>
+                    <li class="nav-link" id="navlink">
+                        <a href="AboutPage.html" class="link">About</a>
+                    </li>
+                    <li class="nav-link" id="navlink"><a href="#contact" class="link">Join</a></li>
+                    <li class="nav-link" id="navlink"><a href="#contact" class="link">Sponsor</a></li>
+                    <li class="nav-link" id="navlink"><a href="#contact" class="link">News</a></li>
+                </ul>
+                <div class="baricon" id="nav-toggler">
+                    <div class="bar1"></div>
```

Comment:
- [ ] Looking at your navigation bar, there is a button missing as we find on the design given. Please add it in your design. Look at the comparison 👇 

Given design
![image](https://user-images.githubusercontent.com/90222110/235301853-1d6e51ed-1f02-4bb9-9c93-9b6117ff770b.png)

Your design
![image](https://user-images.githubusercontent.com/90222110/235301874-b42989bc-c7d6-4dc5-8c89-a0262010e9ec.png)